pkgs/development/python-modules/zeep/default.nix

```diff
@@ -44,21 +45,20 @@ buildPythonPackage rec {
     six
   ];
 
-  # testtools dependency not supported for py3k
-  doCheck = !isPy3k;
-
   checkInputs = [
     tornado
   ];
 
-  buildInputs = if isPy3k then [] else [
+  buildInputs = [
+    aiohttp
+    aioresponses
```

Comment:
I suggest conditionally adding these to `propagatedBuildInputs`:
```
++ lib.optionals isPy3k [ aiohttp aioresponses ]
```

Reply:
Why did you mark this as resolved? Zeep still doesn't work on Python 2 with your proposal.